src/store/data-context.js

```diff
@@ -109,10 +17,39 @@ const DataContext = React.createContext({
 });
 
 export const DataContextProvider = ({ children }) => {
-  const [appState, setAppState] = useState(INITIAL_DATA);
+  const [appState, setAppState] = useState([]);
   const [readOnly, setReadOnly] = useState(false);
   const [addingCard, setAddingCard] = useState(false);
 
+  const getCards = useCallback(async () => {
+    let cards = [];
+
+    try {
+      const response = await axios.get(
+        "https://raw.githubusercontent.com/BrunnerLivio/PokemonDataGraber/master/output.json"
+      );
+
+      const data = response.data;
+
+      for (let i = 0; i < 15; i++) {
+        let card = {
+          id: uuidv4(),
+          caption: data[i].Name,
+          text: data[i].About,
+          checked: false,
+        };
+
+        cards.push(card);
+      }
```

Comment:
This could be done in a functional way, using array methods
Its not necessary, but you can try if you want